### ISSUE TITLE
Suggestion : auto worth calculate

### ISSUE THREAD

**Issue by mmuziek** [mastercake10/uShop#4]
hey i have a suggestion.
mabe if someone sells for example an iron ingot. take the value out of the list and lower it if many people sell it until a bottom line configurable. and for every like 1 minecraft day add the worth back to the item.

for example iron is worth €100 per piece.
i sell 50 iron to the shop and its configurated to take 1% of the price for each sale
€100 - (100 / 1 = € 1) * 50 = €50 = €50 is the new worth.
now after one minecraft day its worth €50.5 because it adds 1 % until it adds up to the worth it was before (€100)

**Comment by ghost** [comment]
Good idea, I just installed the plugin and I do not want to set a price for all items manually!

**Comment by Peda1996** [comment]
How should this be ajustable? On item basis? Or in general? I might be interested in implementing, but i need youre advice first.

**Comment by mmuziek** [comment]
its basically on item basis so
as told above there is a base value for example 100 euros.
if someone sells a lot of that item it lowers the worth by 1% until its almost worthless.
However if someone buys it again the value goes up. increasing it by 1%

**Comment by Peda1996** [comment]
Well, but buying isn't implemented, and therefore we have to find other constraints to work with instead..

**Comment by mmuziek** [comment]
you can change it so it just goes back in time instead of at buying
so for example every hour it goes up 1%

**Comment by Peda1996** [comment]
I don't like this idea, it won't scale on big servers, maybe add the number of players or overall sells to define this % that the price goes up again?
